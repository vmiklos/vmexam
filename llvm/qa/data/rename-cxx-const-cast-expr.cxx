class C
{
public:
    int getValue()
    {
        return 0;
    }
};

int main()
{
    const C* pC = new C();
    const_cast<C*>(pC)->getValue();
}
