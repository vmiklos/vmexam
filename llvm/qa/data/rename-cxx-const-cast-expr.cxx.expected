class D
{
public:
    int getValue()
    {
        return 0;
    }
};

int main()
{
    const D* pC = new D();
    const_cast<D*>(pC)->getValue();
}
